### ISSUE TITLE
remove/add answers automatically

### ISSUE THREAD

**Issue by tavareshugo** [cambiotraining/quarto-course-template#16]
This [blog post](https://nrennie.rbind.io/blog/r-tutorial-worksheets-quarto/) shows a trick for showing/hiding parts of the markdown (which could be applied to exercise answers). 

That solution works for R, but perhaps not quite the same for python-based courses. 
Would be good to explore whether this can be somehow applied to the callout extension, for example having a conditional that looks at the YAML parameters and adds the extra `.content-hidden` class to the callout box.

**Comment by tavareshugo** [comment]
This can also be done with [profiles](https://quarto.org/docs/projects/profiles.html). 
For example: 

```md
::: {.content-hidden when-profile="hide"}
::: {.callout-answer}
The exercise answer
:::
:::
```

If we run with `quarto render` then the answers will be rendered as normal. 
If we run with `quarto render --profile hide` then the answers will be hidden.

**Comment by tavareshugo** [comment]
I've implemented a solution as part of the extension (fe4e9151aa864b812ea8b9f07c59b99ff7219301)

This is [documented here](https://cambiotraining.github.io/quarto-course-template/materials/02-content_guidelines.html#configuring-exercises).

@mvanrongen and @Vicki-H you might be interested in checking this out. You'll have to update the extension in your courses (`quarto update extension cambiotraining/quarto-course-template`) and then edit your `_quarto.yml` if you want to turn on these options (see [example here](https://github.com/cambiotraining/quarto-course-template/blob/fe4e9151aa864b812ea8b9f07c59b99ff7219301/_quarto.yml#L17)).
